cpp/src/parquet/column_reader.h

```diff
@@ -55,6 +56,21 @@ static constexpr uint32_t kDefaultMaxPageHeaderSize = 16 * 1024 * 1024;
 // 16 KB is the default expected page header size
 static constexpr uint32_t kDefaultPageHeaderSize = 16 * 1024;
 
+// \brief DataPageStats is a proxy around format::DataPageHeader and
```

Comment:
The comment seems inaccurate since it does not have access to all fields in the thrift page header.

Reply:
Done.